spec/lib/web_of_science/contributions_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe WebOfScience::Contributions do
+  class WosContributions
```

Comment:
Please use an anonymous class to prevent namespace pollution in other tests.  This has been a problem in our other projects where the same test "setup" gets replicated and changed in different files, but the class definitions persist/interact/conflict.

E.g. 
```ruby
let(:test_class) { Class.new { include WebOfScience::Contributions } } # or described_class
subject(:wos_contributions) { test_class.new }
```

Reply:
Pushed up this change, thanks.